README.md

```diff
@@ -116,11 +102,11 @@ lspconfig.texlab.setup{
 
 ### Example: custom config
 
-To configure a custom/private server, just require `lspconfig/configs` and do
-the same as we do if we were adding it to the repository itself.
+To configure a custom/private server, just 
 
-1. Define the config: `configs.foo_lsp = { … }`
-2. Call `setup()`: `require'lspconfig'.foo_lsp.setup{}`
+1. require `lspconfig/configs`,
+2. Define the config: `configs.foo_lsp = { … }`
```

Comment:
Is that correct? Don't you need to have `local configs = require'lspconfig.configs` for this to work?

Reply:
This whole functions is broken, see https://github.com/neovim/nvim-lspconfig/issues/191

I "fixed this" by silently ignoring the require error, but teto had some concerns. https://github.com/neovim/nvim-lspconfig/pull/475

Reply:
Yeah, I have the strong opinion that any code snippet in a README should be working when copied and pasted (up to necessary customization)...

Reply:
And I don't think this should error.
```lua
local configs = require 'lspconfig.configs'
configs.texlab = {}
```
works fine for me?

Reply:
That's true, the main thing that errors is testing for membership, because the metatable defines new index's to 'require' the module (which doesn't exist when defining your own server)

Reply:
(Meaning, you don't need the `if not ...` guard. If the config already exists, it is just overwritten.)

Reply:
I just went ahead and removed it for now, I'll add it back in #191